test/functional/commands/attribute-e2e-specs.js

```diff
@@ -15,6 +15,23 @@ let caps = {
   platformName: 'Android'
 };
 
+async function waitForElement (strategy, selector) {
```

Comment:
this could all be replaced with `asyncbox`'s `retryInterval`

```js
async function waitForElement (strategy, selector) {
  return retryInterval(10, 3000, async () => {
    await driver.findElOrEls(strategy, selector);
  };
}
```

or set an implicit wait on the driver first.